pkg/eth/ipld_retriever.go

```diff
@@ -366,23 +402,28 @@ func (r *IPLDRetriever) RetrieveReceiptsByBlockHash(hash common.Hash) ([]string,
 
 // RetrieveReceiptsByBlockNumber returns the cids and rlp bytes for the receipts corresponding to the provided block hash
```

Comment:
We may want to note that the CIDs we are returning is not for the Receipts themselves but rather the leaf nodes they are contained within

Reply:
Done